CONTRIBUTING.md

```diff
@@ -24,7 +24,10 @@ For more information, please see STYLEGUIDE.md
 
 # <a name="Attachments"></a>Attachments
 
-GitHub only allows image attachments. Other attachments, for example zipped logs, can be posted in the <a href="http://www.universalmediaserver.com/forum/viewtopic.php?f=14&t=1656">Github attachments thread</a> in the UMS forum.
-Remember to include a link to the attachment in your post here.
+GitHub only allows a few attachment extensions: .png, .gif, .jpg, .docx, .pptx, .xlsx, .txt, .pdf, .zip and .gz.  
```

Comment:
Maybe we can keep it more general so that we don't have to update it every time GitHub updates its formats. Maybe "GitHub supports attaching some files like images and archives."?